tests/testthat/test-derive_vars_dt.R

```diff
@@ -635,4 +635,40 @@ test_that("derive_vars_dt Test 25: Supplying both min/max dates for highest_impu
   expect_dfs_equal(actual, expected, keys = c("ASTDT", "ASTDTF"))
 })
 
+# data for Test 26 ----
+
+date <- tibble::tribble(
+  ~XXSTDTC, ~ASTDTF,
+  "2019-07-18T15:25:40", NA_character_,
+  "2019-07-18", NA_character_,
+  "2019-02", NA_character_,
+  "2019", NA_character_,
+  "2019---07", NA_character_
+)
```

Comment:
The input data should be created in the test (unless it is used for several tests).

To avoid repetition you could define `date <- select(expected_output, -ASTDT)`.

Reply:
Thanks, @bundfussr . Addressed.